setup.py

```diff
@@ -2,15 +2,8 @@
 # -*- coding: utf-8 -*-
```

Comment:
fyi this is linked to the follow issue:  https://github.com/pacifica/python-jsonpath2/issues/46